src/lib/localStorage.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { getLocalStorageItem, setLocalStorageItem, removeLocalStorageItem } from "./localStorage";
+
+describe("localStorage tests", () => {
+  beforeEach(() => {
+    localStorage.clear();
+    jest.clearAllMocks();
+  });
+
+  test("getLocalStorageItem not existing", () => {
+    expect(getLocalStorageItem("test")).toBeUndefined();
+  });
+
+  test("getLocalStorageItem existing", () => {
+    localStorage.setItem("test", '{"data":{"field1":"hello","field2":"world"}}');

```

Comment:
something like this would be a little more neat:

```
 const testData = { field1: "hello", field2: "world" };
    localStorage.setItem("test", JSON.stringify({ data: testData }));
    expect(getLocalStorageItem("test")).toEqual(testData);
```

Reply:
changed as you suggested